AllReadyApp/Web-App/AllReady/Models/Activity.cs

```diff
@@ -7,6 +7,13 @@ namespace AllReady.Models
 {
     public class Activity
     {
+        public Activity()
+        {
+            Tasks = new List<AllReadyTask>();
```

Comment:
Hmm. That is one C#6 feature that I'm not totally sold on yet.  I will change it though